Yoast/javascript
# Add InsightCard component

## Summary


This PR can be summarized in the following changelog entry:

* Adds an InsightCard component.

## Relevant technical choices:

* Adds InsightCard component for the Insight modal

## Test instructions

This PR can be tested by following these steps:

* See test instructions for [the corresponding branch on Premium](https://github.com/Yoast/wordpress-seo-premium/pull/3121)

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes P1-228
